janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/KeySliceQuery.java

```diff
@@ -79,6 +79,6 @@ public boolean subsumes(KeySliceQuery oth) {
 
     @Override
     public String toString() {
-        return String.format("KeySliceQuery(key: %s, start: %s, end: %s, limit:%d)", key, getSliceStart(), getSliceEnd(), getLimit());
+        return "KeySliceQuery(" + key + ")[" + getSliceStart() + "," + getSliceEnd() + ")@" + getLimit();
```

Comment:
Why is the `[` closed by `)`?

Reply:
This is to indicate this is a half-closed interval: [sliceStart, sliceEnd) means sliceStart is inclusive while sliceEnd is exclusive. The old way is clear but a bit lengthy.